src/sleep/sleep.c

```diff
@@ -366,6 +366,9 @@ static int freeze_thaw_user_slice(const char **method) {
         if (r < 0)
                 return log_debug_errno(r, "Failed to open connection to systemd: %m");
 
+        /* Wait for 1.5 seconds at maximum for freeze operation */
+        sd_bus_set_method_call_timeout(bus, 1500 * USEC_PER_MSEC);
```

Comment:
hmm, on an overloaded system 1.5s sounds a bit too little. let's make this 5s?

also, please cast result to (void), given you knowingly ignore any errors here. static checkers care for this.